backend/src/Infrastructure/Logging/ParameterSafeCommand.php

```
@@ -0,0 +1,41 @@
+declare(strict_types=1);
+
+namespace App\Infrastructure\Logging;
+
+use Yii;
+use yii\db\Command;
+use yii\db\Exception;
+
+/**
+ * Keeps bind parameter values out of query, profile, and DB exception messages.
+ */
+final class ParameterSafeCommand extends Command
```

PHPStan (2.2.7)

[error] Class App\Infrastructure\Logging\ParameterSafeCommand extends unknown class yii\db\Command.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)